samples/cpp/tutorial_code/ImgTrans/CannyDetector_Demo.cpp

```diff
@@ -5,10 +5,11 @@
  */
 
 #include "opencv2/imgproc.hpp"
-#include "opencv2/imgcodecs.hpp"
 #include "opencv2/highgui.hpp"
+#include <iostream>
 
 using namespace cv;
+using namespace std;
```

Comment:
Please remove this line to avoid this error on Apple Mac:
```
/build/precommit_macosx/opencv/samples/cpp/tutorial_code/ImgTrans/CannyDetector_Demo.cpp:39:71: error: reference to 'ratio' is ambiguous
    Canny( detected_edges, detected_edges, lowThreshold, lowThreshold*ratio, kernel_size );
                                                                      ^
/build/precommit_macosx/opencv/samples/cpp/tutorial_code/ImgTrans/CannyDetector_Demo.cpp:21:5: note: candidate found by name lookup is 'ratio'
int ratio = 3;
    ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ratio:234:29: note: candidate found by name lookup is 'std::__1::ratio'
class _LIBCPP_TYPE_VIS_ONLY ratio
                            ^
1 error generated.
```